sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/screens/home/HomeViewModel.kt

```diff
@@ -12,39 +15,33 @@ import javax.inject.Inject
 
 @HiltViewModel
 class HomeViewModel @Inject constructor(
-    private val getModelsUseCase: GetModelsUseCase,
-    private val isFirstTimeLaunchPreferencesUseCase: IsFirstTimeLaunchPreferencesUseCase,
-    private val updateFirstTimeLaunchPreferencesUseCase: UpdateFirstTimeLaunchPreferencesUseCase,
-    dispatchers: DispatchersProvider
-) : BaseViewModel(dispatchers) {
+    dispatchersProvider: DispatchersProvider,
+    getModelsUseCase: GetModelsUseCase,
+    isFirstTimeLaunchPreferencesUseCase: IsFirstTimeLaunchPreferencesUseCase,
+    updateFirstTimeLaunchPreferencesUseCase: UpdateFirstTimeLaunchPreferencesUseCase,
```

Comment:
Just curious, but do we have any convention on how we group the ViewModel's attributes? As I saw from the previous code, we are placing `dispatchers: DispatchersProvider` at the very bottom, but we are moving it to the top now. Should we always group the use cases at the top like the same approach as the old code? 🤔

Reply:
@minhnimble Good question 👍 I can't find an actual convention for this, however, from [this code sample](https://kotlinlang.org/docs/coding-conventions.html#class-properties-and-parameters), I think we should put `non val` properties at the end 👍 

So in final, the order could be like this 💭 
- public val
- non public val
- private val
- non private val

Reply:
@luongvo FYI, a Discussion is created for this convention here https://github.com/nimblehq/android-templates/discussions/451 🙏